runtimes/pythonrt/Makefile

```diff
@@ -17,7 +17,7 @@ test-py:
 
 test-go:
 	golangci-lint run ./...
-	go test -v
+	go test $(SHORT) -v
```

Comment:
where is this coming from?

Reply:
You can set environment variable to run only short tests, this disables `Test_createVEnv`  which takes a long time.